packages/swirl-components/src/components/swirl-form-control/swirl-form-control.tsx

```diff
@@ -248,6 +249,11 @@ export class SwirlFormControl {
             <LabelTag class="form-control__label" onClick={this.onLabelClick}>
               <span class="form-control__label-text" id={this.labelId}>
                 {this.label}
+                {this.secondaryLabel && this.labelPosition === "outside" && (
```

Comment:
**question:** Why are we not displaying the secondaryLabel when the labelPosition is `inside`? Is it because the "primary" label is already subdued?

Reply:
i have no design for that. so i am assuming we only display it outside :)